db/migrate/20170706201604_add_source_id_to_user.rb

```diff
@@ -0,0 +1,10 @@
+class AddSourceIdToUser < ActiveRecord::Migration
+  def change
+    add_reference :users, :source, index: true, foreign_key: true
+    User.find_each do |u|
+      s = Source.where(user_id: u.id).first
+      u.source_id = s.id
```

Comment:
I'm not sure if all users in the database have a source (should have)... please test with a dump from live